package.json

```diff
@@ -40,6 +40,7 @@
     "codemirror": "^5.65.9",
     "cron-validator": "^1.3.1",
     "cronstrue": "^2.21.0",
+    "dayjs": "^1.11.7",
```

Comment:
This is no longer needed, if the below suggestion works for you

Reply:
still used